docker/postgres/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM postgres:10.15
+FROM postgres:12.13
 
 RUN apt-get update \
     && apt-get install --no-install-recommends -y \
-    postgis postgresql-10-postgis-2.4 postgresql-10-postgis-2.4-scripts
```

Comment:
Those scripts are not required anymore?

Reply:
Nah, I think I shouldn't have actually removed it. I forgot to put it back after testing something else and didn't notice because everything still seemed okay at a surface, but the `postgresql-12-postgis-3` package does still instruct to install the equivalent scripts.